Lesson00_PreWork/Lesson00_PreWork/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace Lesson00_PreWork
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            View.DisplayMethod();
+        }
+    }
+
+    public class Controller
+    {
+        public Controller()
+        {
+        }
+
+        public static void LoopNum(int RepeatingChar)
+        {
+            int RepeatNum = 10;
+            for (int i = 0; i < RepeatNum; i++)
+            {
+                Console.Write(Model.RepeatingChar);
+            }
+        }
+    }
+
+    public class Model
+    {
+        public static int RepeatingChar { get; set; }
+
+        public Model (int repeatingChar)
+        {
+            RepeatingChar = repeatingChar;
+        }
+    }
+
+    class View
+    {
+        public static void DisplayMethod()
+        {
+            Model myModel = new Model(8);
+            Controller.LoopNum(Model.RepeatingChar);
```

Comment:
Your controller is responsible for driving the flow of your application, not the view. The Controller should be passing the model to the view, but here you are telling the controller to draw the view. A good way to think about this is, the view should be the only class with Console.Write and Console.WriteLine calls in it.